misc/python/materialize/optbench/sql.py

```diff
@@ -70,9 +71,20 @@ def __init__(
         port: int,
         host: str,
         user: str,
+        password: Optional[str],
+        require_ssl: bool,
     ) -> None:
         logging.debug(f"Initialize Database with host={host} port={port}, user={user}")
-        self.conn = pg8000.connect(host=host, port=port, user=user)
+
+        if require_ssl:
+            ssl_context = ssl.create_default_context()
+            ssl_context.verify_mode = ssl.CERT_REQUIRED
```

Comment:
Do you need to set `ssl.CERT_REQUIRED` here? It seems to be the default setting with `ssl.create_default_context()` according to: https://docs.python.org/3/library/ssl.html#ssl.create_default_context.

Reply:
Done.